### ISSUE TITLE
Can I compile a 32-bit dll?

### ISSUE THREAD

**Issue by dqi1999** [webui-dev/webui#286]
Can I compile a 32-bit dll?

**Comment by jinzhongjia** [comment]
of course, I don't know how to that with gcc,but you can use this command `zig build -Dtarget=x86-linux -Dis_static=false -Denable_tls=false` to build dynamic library for x86(32bits) windows (if you want to enable tls support, use `-Denable_tls=true`)

**Comment by jinzhongjia** [comment]
After running `zig build -Dtarget=x86-linux -Dis_static=false -Denable_tls=false`, you can find the dynamic library in the directory `zig-out/lib`

**Comment by dqi1999** [comment]
thank you!But I do not use zig to compile, testing for half a day did not get the dynamic link library, you can also release a 32-bit dynamic link library when you publish?

**Comment by jinzhongjia** [comment]
Sorry, I am not responsible for the webui of the C library. Need to ask @hassandraga @ttytm  about this.

**Comment by dqi1999** [comment]
Thank you!

**Comment by dqi1999** [comment]
zig compiled successfully using the latest version of webui；use cmd: zig build -Dtarget=x86-linux -Dis_static=false -Denable_tls=false,But the output is a file libwebui.so;

**Comment by jinzhongjia** [comment]
Yes, it seems that there are some strange problems with dynamic link building windows, I fixed them normally

**Comment by jinzhongjia** [comment]
this pr should fix this problem:https://github.com/webui-dev/webui/pull/289
